layers/cmdline.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+from exporters.to_svg import ToSvg
+from exporters.to_excel import ToExcel
+from core import Layer
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Export an Att&ck layer as a svg image or excel file')
```

Comment:
Att&ck should be ATT&CK as per "how should I reference the name ATT&CK" in the [ATT&CK FAQ](https://attack.mitre.org/resources/faq/):
> Always capitalize "ATT&CK" to distinguish it from the surrounding text.

Note that writing `ATT&CK` as `attack` is acceptable in source code, just not documentation or UIs.

Reply:
Also, should probably say "ATT&CK *Navigator* layer" to provide user additional context.

Reply:
Good catch.